src/io.rs

```diff
@@ -133,19 +133,28 @@ use std::path::{Path, PathBuf};
 impl Deserialize for crate::Texture2D {
     fn deserialize(path: impl AsRef<std::path::Path>, raw_assets: &mut RawAssets) -> Result<Self> {
         let path = raw_assets.match_path(path.as_ref())?;
+        let extension = path
+            .extension()
+            .map(|e| e.to_str().unwrap())
+            .unwrap_or("image")
+            .to_string();
         #[allow(unused_variables)]
         let bytes = raw_assets.get(&path)?;
 
-        #[cfg(not(feature = "image"))]
-        return Err(Error::FeatureMissing(
-            path.extension()
-                .map(|e| e.to_str().unwrap())
-                .unwrap_or("image")
-                .to_string(),
-        ));
+        if cfg!(feature = "svg") && "svg" == extension {
```

Comment:
I don't think you need `cfg!(feature = "svg")`

Reply:
Ah, that is true. Should probably have noticed this one sooner 🙂